pandas/tests/io/formats/test_to_csv.py

```diff
@@ -6,6 +8,18 @@
 
 class TestToCSV(object):
 
+    def test_to_csv_defualt_encoding(self):
+        # GH17097
+        df = DataFrame({'col': [u"AAAAA", u"ÄÄÄÄÄ", u"ßßßßß", u"聞聞聞聞聞"]})
+
+        with tm.ensure_clean('test.csv') as path:
+            if pd.compat.PY2:
```

Comment:
Add a comment about why we're raising an error and what should be done to avoid this error (i.e. the `encoding` argument parameter should be correct).